NDTensors/src/abstractarray/set_types.jl

```diff
@@ -11,7 +11,7 @@ TODO: Use `Accessors.jl` notation:
 # TODO: Delete this when we change to using a
 # `FillArray` instead. This is a stand-in
 # to make things work with the current design.
-function SetParameters.set_ndims(numbertype::Type{<:Number}, ndims)
+function TypeParameterAccessors.set_ndims(numbertype::Type{<:Number}, ndims)
```

Comment:
Potentially this should be in `TypeParametersAccessors`

Reply:
I would lean towards leaving this here since it will get removed later anyway, since it only exists to serve the quirky design of the `Diag` type.